etc/test-helper-environment-template.js

```diff
@@ -0,0 +1,23 @@
+(function(require) {
+  var simple = require('simple-mock');
+
+  var requireAccess = simple.stub();
+  var requireRole = simple.stub();
+  var requireUser = simple.stub();
+  var channel = simple.stub();
+  var access = simple.stub();
+  var role = simple.stub();
+
+  var customActionStub = simple.stub();
+
+  return {
+    requireAccess: requireAccess,
+    requireRole: requireRole,
+    requireUser: requireUser,
+    channel: channel,
+    access: access,
+    role: role,
+    customActionStub: customActionStub,
+    syncFunction: %SYNC_FUNC_PLACEHOLDER%
+  };
+});
```

Comment:
The parentheses here are a bit confusing to an uninitiated reader. Might be worth explaining why they're needed here or where the template is used.

Wherever that comment goes, it may be useful to explain that the placeholder should be/is replaced with the actual sync function.